app/Http/Livewire/ArgumentList.php

```diff
@@ -74,7 +74,9 @@ public function render(): View
 
         $query = $sortField->applySort($query);
 
-        $arguments = $query->paginate(15);
+        $arguments = $query
+            ->paginate(15)
+            ->setPath(action(RfcDetailController::class, $this->rfc));
```

Comment:
This is how you fixed it? I had no idea how to do it 🤔

Reply:
Yeah, actually, it took me some time to understand what was going on. It turns out that we need to set the path if we are going to use a custom pagination component because it works fine with the default component.

https://laracasts.com/discuss/channels/laravel/error-in-paginator-link-caused-by-livewiremessageservices-allpage2
https://github.com/livewire/livewire/discussions/3079